app/templates/gentelella/admin/event/tickets/access_codes_create.html

```diff
@@ -227,9 +227,8 @@
 
         var host = (document.URL.split("//")[1].split("/"))[0];
         $("#code").change(function(){
-            $("#link").val(document.URL.split("/")[0]+"//"+host+"/e/"+"{{ event.identifier }}"+"/?code="+$(this).val());
+            $("#link").val(document.URL.split("/")[0]+"//"+host+"/e/"+"{{ event.identifier }}"+"/?access_code="+$(this).val());
```

Comment:
@Jatinbalodhi96  This is for access and similarly for discount you have another link. How will this combine both access and discount  together in a single link?
like: "**acode=VIP&dcode=888**". You want them both on a single link.